### PR TITLE
Include af_packet zeekctl plugin

### PR DESCRIPTION
**Pull request by awelzel**
* This imports the version from the plugin into zeekctl
* Adds support for setting lb_method = af_packet, rather than going through lb_method custom. Seems reasonable now that it's available in-tree.

Quirkiness: With the lb_method = "custom" interface prefix/suffix idea, there's actually a dependency between lb_custom.py and af_packet.py, so I prefixed the later with `zzz_` during the import and made the imports predictable rather than opening the plugin dependencies topic within zeekctl, too. With direct support for lb_method = af_packet, there shouldn't be much reason to use lb_custom with af_packet.

### CODE REVIEW

**Comment by awelzel** [comment]
I have tested this with the following worker entry:
```
[worker-1]
type=worker
host=192.168.123.132
lb_method = af_packet
lb_procs = 4
interface=enp6s0
af_packet_fanout_id=43
```

And that seems to do what it should. I've also verified using lb_method = custom and setting and prefixing interface directly works.

The zzz_af_packet.py file will override af_packet.py shipped by an external af_packet plugin.

File: ZeekControl/plugins/zzz_af_packet.py [hunk 1]
```diff
@@ -11,12 +11,27 @@ def pluginVersion(self):
...
-            if nn.type == "worker" and nn.interface.startswith("af_packet::") and nn.lb_procs:
-                return True
+            if nn.type != "worker" or not nn.lb_procs:
+                continue
+
+            if nn.interface.startswith("af_packet::"):
+                result = True
+
+            if nn.lb_method == "af_packet":
+                nn.interface = "af_packet::%s" % nn.interface
```
Comment (rsmmr) [inline ZeekControl/plugins/zzz_af_packet.py:31]: Does this need to guard against an `af_packet::` prefix being present already?

**Comment by rsmmr** [comment]
> The zzz_af_packet.py file will override af_packet.py shipped by an external af_packet plugin.

I'm wondering if we could instead now just reject an external `af_packet` plugin, and report an error message telling people to uninstall it.

**Review by rsmmr — APPROVED** [review]
Looks reasonable to me, although I don't think I fully understand the dependency (and its implication) between `lb_custom` and `lb_af_packet`.

**Comment by awelzel** [comment]
> > The zzz_af_packet.py file will override af_packet.py shipped by an external af_packet plugin.
> 
> I'm wondering if we could instead now just reject an external `af_packet` plugin, and report an error message telling people to uninstall it.

Meaning uninstall the af_packet plugin completely or just the .py file it ships? Either way, this seems cause errors in scenarios where an external af_packet plugin is used (and the one in Zeek disabled) - a scenario I understand we don't yet want to drop support for..

File: ZeekControl/plugins/zzz_af_packet.py (hunk 1 above)
Comment (awelzel) [reply ZeekControl/plugins/zzz_af_packet.py:31]: Yeah, we could be more friendly here and say `unexpected af_packet:: interface prefix with lb_method "af_packet"` or some such.

**Comment by awelzel** [comment]
For reference, the actual message when installing an external AF_Packet plugin *and* having zeekctl provide it, is as follows:
```
zeek@ubuntu-01:/opt/zeek/etc$ zeekctl
Warning: failed to load plugin af_packet (prefix af_packet) due to plugin af_packet (prefix af_packet) having the same prefix
```

It's not super helpful, but things continue to work (the internal zzz_af_packet.py is used). Given the few people that concerns, I'm not worried about the cosmetics too much.

**Review by ckreibich — APPROVED** [review]

**Pull request merged**